src/student/components/studentWalletViewComponents/walletSummary/WalletSummary.tsx

```diff
@@ -23,8 +23,14 @@ const WalletSummary: React.FC<WalletSummaryProps> = ({ data }) => {
   //   });
   // };
 
-  const liquidityPercentage = (data.availableCoins / data.totalBalance) * 100;
-  const investmentPercentage = (data.investedCoins / data.totalBalance) * 100;
+  const liquidityPercentage = data.totalBalance
+    ? (data.availableCoins / data.totalBalance) * 100
+    : 0;
+
+  const investmentPercentage = data.totalBalance
+    ? (data.investedCoins / data.totalBalance) * 100
+    : 0;
+
```

Comment:
Como los usas mas de una vez, aca podrias descomponer data en los tres valores que estas usando,

Primero, arriba de todo pondrias:
```ts
const { totalBalance, availableCoins, investedCoins } = data;
```

Y despues en el codigo usas
```suggestion
  const liquidityPercentage = totalBalance ? (availableCoins / totalBalance) * 100 : 0;

  const investmentPercentage = totalBalance ? (investedCoins / totalBalance) * 100 : 0;
```

Reply:
Bien, banco banco